### PR TITLE
[Module Port - Theme] Bloody Hell Mate

### PR DESCRIPTION
**Pull request by valentinegb**
As requested by Westly/Alicia.

### CODE REVIEW

**Comment by CanadaHonk** [comment]
Hey, can you ensure (screenshot / etc) you got permission for the port? Just to check, thanks.

File: modules/themes/bloodyHellMate.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+const version = "0";
+
+let style;
+
+let obj = {
+  onImport: async () => {
+    goosemodScope.showToast("onImport()");
```
Comment (CanadaHonk) [inline modules/themes/bloodyHellMate.js]: This isn't needed.

File: modules/themes/bloodyHellMate.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    );
+  },
+
+  remove: async () => {
+    style.remove();
+  },
+
+  name: "Bloody Hell Mate",
+  description: "Makes everything bloody red, mate!",
+  author: ["MrDragonBoi", "Liam The Protogen"],
```
Comment (CanadaHonk) [inline modules/themes/bloodyHellMate.js:22]: Still unsure whether we should add porter as an author.

**Review by CanadaHonk — CHANGES_REQUESTED** [review]

File: modules/themes/bloodyHellMate.js (hunk 1 above)
Comment (valentinegb) [reply modules/themes/bloodyHellMate.js]: Ack I was testing something, forgot about that sorry.

File: modules/themes/bloodyHellMate.js (hunk 2 above)
Comment (valentinegb) [reply modules/themes/bloodyHellMate.js:22]: This is true... I was kinda in a rush to make this, sorry

**Comment by CanadaHonk** [comment]
> Hey, can you ensure (screenshot / etc) you got permission for the port? Just to check, thanks.

Got evidence from OP, looks good to continue permission wise.
Screenshot

File: modules/themes/bloodyHellMate.js (hunk 2 above)
Comment (CanadaHonk) [reply modules/themes/bloodyHellMate.js:22]: No worries. More of a reminder / call to self that we should think about this in the future.

File: modules/themes/bloodyHellMate.js (hunk 1 above)
Comment (CanadaHonk) [reply modules/themes/bloodyHellMate.js]: No worries. Solving via edit myself now.

**Comment by CanadaHonk** [comment]
Alright looks good now after self edit.

**Review by CanadaHonk — APPROVED** [review]

**Pull request merged**